ciao-controller/internal/datastore/datastore.go

```diff
@@ -1453,10 +1405,23 @@ func (ds *Datastore) GetTenantCNCISummary(cnci string) ([]types.TenantCNCI, erro
 			continue
 		}
 
+		var IPAddress string
+		var MACAddress string
+
+		if t.CNCIID != "" {
```

Comment:
If t.CNCIID=="" then an empty TenantCNCI will be returned. Is there value in the that? Shouldn't we error out or not include the CNCI in the slice returned?